### DISCUSSION TITLE
Problème de synchronisation du serveur wikijs avec Active Directory

### DISCUSSION THREAD

**Discussion by wilmakmarie** [requarks/wiki#7078] [Help / Questions]
Bonjour,
Je rencontre des difficultés depuis plusieurs jours pour synchroniser mon serveur WikiJS avec mon domaine, ce qui empêche les utilisateurs de se connecter. J'aurais besoin de vos éclaircissements, compétences et assistance pour résoudre ce problème. Pour vous aider à comprendre ma situation, voici les détails de ma configuration actuelle au niveau de mon serveur WikiJS.

Nom daffichage : LDAP / Active Directory
LDAP URL : ldap://adresse_du_serveur_AD:389
Admin Bind DN: CN=Administrateur,CN=Users,DC=domaine,DC=lan
Admin Bind Credentials: Mot_de_passe_administrateur_du_domaine
Search Base: OU=Users,OU=MMJ2,DC=domaine,DC=lan
Search Filter: (sAMAccountName={{username}}))

Use TLS= Non Activé
Verify TLS Certificate= Non Activé

Unique ID Field Mapping: sAMAccountName
Email Field: mail
Display Name Field Mapping: displayName

Pour finir j'ai activé l'auto-inscription

Merci de me répondre.

Cordialement,

**Comment by ghost** [comment]
Salut,

Je rencontre le même soucis ... j'ai suivi plusieurs tuto mais je n'arrive pas à le mettre en place.
J'ai également utilisé l'outil de DEBUG LDAP dans WIKIJS il me dit LDAP OK.

Peut on avoir plus d'aide ?

---

Hi,

I'm facing the same issue... I've followed several tutorials but I can't get it to work. I also used the LDAP DEBUG tool in WIKIJS and it tells me LDAP OK.

Can we get more help?
